src/measure_theory/decomposition/signed_hahn.lean

```diff
@@ -41,16 +41,23 @@ variables {α : Type*}
 
 open set filter
 
+-- TODO: generalize to `complete_boolean_algebra`
+lemma set.Union_diff_Union_nat_lt (f : ℕ → set α) :
```

Comment:
This lemma is almost exactly `set.Union_disjointed`

Reply:
Which suggests to me that you probably want to use `set.disjointed` in lots of your lemmas.

Reply:
I've now used `set.disjointed` instead, and they are certainly what I had in mind in the first place :)